examples/placegoose/wrangler.toml

```diff
@@ -0,0 +1,50 @@
+name = "placegoose"
+compatibility_date = "2024-11-15"
+
+compatibility_flags = [ "nodejs_compat" ]
+
+assets = { directory = "public" }
```

Comment:
this is awesome. we have another sample app coming with static asset bindings too, using it in a slightly different way. probably could do a separate post on how we've set up static assets in both of those projects